packages/apify/src/storages/dataset.ts

```diff
@@ -262,8 +261,7 @@ export class Dataset {
      * }
      * ```
      */
-    async getInfo<T extends Dictionary = Dictionary>(): Promise<T> {
-        // @ts-ignore
+    async getInfo() {
```

Comment:
would be good to specify the type explicitly as this is public api

Reply:
It felt redundant since TS infers the type anyways,but I can add it back! Can we setup the eslint rule then that always enforces return types?

Reply:
for public methods yes. the thing is that with inference, the upstream can change return value and introduce a BC. (it probably won't happen with this one, more like a general rule of thumb)